repose-aggregator/components/filters/openstack-identity-v3-filter/src/main/resources/META-INF/schema/config/openstack-identity-v3.xsd

```diff
@@ -98,6 +98,18 @@
             </xs:annotation>
         </xs:attribute>
 
+        <!--TODO: Rather than using an attribute, create a cache element like in Keystone v2.
+                  Deprecate the current cache attributes, and move them to the element. -->
+        <xs:attribute name="atom-feed-ids" type="AtomFeedIdsType" use="optional">
```

Comment:
I don't love this way of configuring Atom feeds to use for invalidation. In Keystone v2, we use a sequence of elements, the values are which are the Atom feed IDs. Doing that here would be less consistent with the rest of the Identity v3 configuration; that is, all of the cache configuration is currently done in attributes.

One particularly important observation to make is that using a `xs:list` means that Atom feed IDs will not be able to contain whitespace.

I've thought about moving the cache stuff into an element like in Keystone v2, but we'd probably want to deprecate the old attributes and support both for now.